mantidproject/mantid
# Fix dragging and dropping files into Mantid on OSX

**This needs both a Window/Linux and an OSX tester**

This adds a work around to a qt bug in Qt4 that breaks drag and drop on OSX.

**To test:**
 - With an OSX machine `>=10.10` test that you can reproduce the bug.
 - Apply the changes and check they fix the bug for OSX.
 - Check that drag & drop is not broken on Windows or Linux.
 - Also check the following places:
   - Dragging & dropping into the Load dialog works on OSX and Windows/Linux.
   - Dragging & dropping into the Python window works on OSX and Windows/Linux.

Fixes #20643 

**Release Notes** 
See the ui release notes for 3.11

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
